packages/broker/src/plugins/operator/closeExpiredFlags.ts

```diff
@@ -0,0 +1,33 @@
+import { EthereumAddress, Logger } from '@streamr/utils'
+import { ContractFacade, SponsorshipResult } from './ContractFacade'
+
+const logger = new Logger(module)
+
+export const closeExpiredFlags = async (
+    maxFlagAgeSec: number,
+    operatorContractAddress: EthereumAddress,
+    contractFacade: ContractFacade
+): Promise<void> => {
+    logger.info('running closeExpiredFlags')
+
+    const minFlagStartTime = Math.floor(Date.now() / 1000) - maxFlagAgeSec
+
+    const sponsorships = (await contractFacade.getSponsorshipsOfOperator(operatorContractAddress))
+        .map((sponsorship: SponsorshipResult) => sponsorship.sponsorshipAddress)
+    if (sponsorships.length === 0) {
+        logger.info('no sponsorships found')
+        return
+    }
+    logger.info(`found ${sponsorships.length} sponsorships`)
```

Comment:
I would log this at `debug` level. Most of the time user doesn't need to know this, only when something is wrong and they want to look into it (hence they enable `debug` level)

```suggestion
    logger.debug(`Found ${sponsorships.length} sponsorships`)
    if (sponsorships.length === 0) {
        return
    }
```